.github/workflows/example.yml

```diff
@@ -72,7 +96,12 @@ jobs:
           # Just check that the root endpoint returns a success status (-f flag).
           command: curl -sSfLi ${{ steps.oc-new-app.outputs.route }}
 
+      - name: Delete Image Stream
+        if: ${{ matrix.image-source == 'IMAGE_STREAM' && always() }}
```

Comment:
why `always()` ?

Reply:
oh i recall now there's an implicit `success()` call in `if`.

Reply:
yeah